android/libraries/rib-base/src/main/java/com/badoo/ribs/core/Interactor.kt

```diff
@@ -50,7 +53,7 @@ abstract class Interactor<C : Parcelable, Content : C, Overlay : C, V : RibView>
     val isAttached: Boolean
         get() = behaviorRelay.value == ACTIVE
 
-    internal var tag = "${this::class.java.name}.${UUID.randomUUID()}"
+    internal var tag = this.savedInstanceState?.getString(KEY_TAG) ?: "${this::class.java.name}.${UUID.randomUUID()}"
```

Comment:
Didn't get the reason of this changes.
If savedInstanceState is null you will use `${this::class.java.name}.${UUID.randomUUID()}`
Than you will save it to bundle and on next attach you will use same tag.

So why do we need to make all this, if we can just use: `${this::class.java.name}.${UUID.randomUUID()}` for all cases?

Reply:
Tags are used for requestcode based actions (permission, activity launching, dialogs). You need to restore existing tag for ensured delivery of e.g. activity result.